website/docs/docs/build/metrics-overview.md

```diff
@@ -227,15 +227,21 @@ metrics:
 
 ## Filters
 
-A filter is configured using Jinja templating. Use the following syntax to reference entities, dimensions, and time dimensions in filters:
+A filter is configured using Jinja templating. Use the following syntax to reference entities, dimensions, time dimensions, or metrics in filters:
+
 
 ```yaml
-filter: |
-  {{ Entity('entity_name') }} 
-filter: |
+filter: | 
+  {{ Entity('entity_name') }}
+
+filter: |  
   {{ Dimension('primary_entity__dimension_name') }}
-filter: |
+
+filter: |  
   {{ TimeDimension('time_dimension', 'granularity') }}
+
+filter: |  
+  {{ Metric('metric_name', group_by=['Entity']) }}
```

Comment:
I would update this to say `entity_name` instead of `Entity` - currently it looks like we might want the `Entity(...)` object syntax, but we just want the name.